apps/backend/src/libs/modules/config/libs/types/environment-schema.type.ts

```diff
@@ -1,4 +1,8 @@
 import { type AppEnvironment } from "~/libs/enums/enums.js";
+import {
+	Algorithm,
+	ExpirationTime,
+} from "~/libs/modules/token/libs/types/types.js";
```

Comment:
```suggestion
} from "~/libs/modules/token/token.js";
```

Reply:
When I try to import from token.ts where i wrote an export for these types at the end of file (exporting from token/libs/types/types.ts), the program fails:

<img width="866" alt="Screenshot 2024-08-19 at 12 06 56" src="https://github.com/user-attachments/assets/362fad17-ff3b-435d-b4c1-ca166ead4c4e">

Could this be a circular dependancy? Because it works if i import the types from "~/libs/modules/token/libs/types/types.js". Should I keep the import as it is to fix the issue?